test/pg-cdc/mzcompose.py

```diff
@@ -12,7 +12,7 @@
 
 SERVICES = [
     Materialized(volumes_extra=["secrets:/share/secrets"]),
-    Testdrive(),
+    Testdrive(default_timeout='20s'),
```

Comment:
Did the test start failing from this change?

Reply:
Oh whoops--accidental from debugging. Will remove. The test in my other branch did not run into this issue.